flow/util/template_filters.py

```diff
@@ -356,3 +356,13 @@ def get_account_name(environment, required=False):
                 file=sys.stderr,
             )
             _GET_ACCOUNT_NAME_MESSAGES_SHOWN.add(env_name)
+
+
+def format_operation_name(operation_name, project):
```

Comment:
Rather than calling this filter in the Jinja templates, could we call it as a pre-processing step someplace like here, replacing `group_name` with something like `format_operation_name(group_name)` where this becomes a method declared inline inside of `FlowProject.print_status`? https://github.com/glotzerlab/signac-flow/blob/50bc29eec23b8bb2c8c277b10dfefd9c7afbdc03/flow/project.py#L3100

My goal is to minimize the complexity of the Jinja templates, at the cost of making `print_status` have another helper function (it already has several).